### PR TITLE
Clarify platform differences in time picker

### LINKED ISSUE #869: TimePicker control displays seconds, but does not allow seconds to be entered?
New to .Net MAUI and was hoping the TimePicker would allow the hours / minutes / seconds to be selected.

The documentation states that "a [standard](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings/) or [custom](https://docs.microsoft.com/en-us/dotnet/standard/base-types/custom-date-and-time-format-strings/) .NET formatting string may be used", implying that all of the time fields (hours, minutes, seconds, milliseconds) could be displayed or selected.  However, when the following code is used:

```
<TimePicker 
   Time="4:15:26" 
   Format="HH:mm:ss"
/>
```

**On the Android simulator:**
- The hours, minutes and seconds are displayed
- When the control is clicked on, only hours and minutes can be selected and an AM / PM selector is displayed (even though 24 hour format specifiers for hours are used, i.e.: hours range from 0-23, but emulator only allows 1-12 to be selected).

**On the Windows Machine simulator**
- The hours and minutes are displayed (seconds are not).
- When the control is clicked on, only hours and minutes can be selected.

I haven't tried the on a Mac / iOS as I do not have a machine to test.

One can only infer that when using the TimePIcker control, only hours and minutes can be selected -- regardless of the Format used.   I had mistakenly assumed that if the format display showed hours, minutes and seconds, the control would allow hours, minutes and seconds to be selected.   Should this be mentioned in the documentation?  I think it might be a good idea.

On another note - is there any TimeSpanPicker available in .Net MAUI?  I have a use case where I need to select a number of days, hours, minutes and seconds -- I guess I could use 4 sliders, but again, for TimeSpan data, it would be great if the docs suggested other resources.

Thanks in advance.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 159cee98-fe87-2242-400a-9ee4f8b3bb8b
* Version Independent ID: 159cee98-fe87-2242-400a-9ee4f8b3bb8b
* Content: [TimePicker - .NET MAUI](https://docs.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker)
* Content Source: [docs/user-interface/controls/timepicker.md](https://github.com/dotnet/docs-maui/blob/main/docs/user-interface/controls/timepicker.md)
* Product: **dotnet-mobile**
* Technology: **dotnet-maui**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

### PR DESCRIPTION
**Pull request by adegeo**
- Added a link for the handlers doc
- Added a **Platform differences** section.

This needs info about macOS

Fixes #869

@davidbritch please edit this PR and add info about mac.

### CODE REVIEW

File: docs/user-interface/controls/timepicker.md [hunk 1]
```diff
@@ -67,3 +67,25 @@ However, this is not recommended. Depending on the setting of the `Format` prope
...
+
+On Android, the `Format` property is respected and displayed by the control. However, when the picker control is shown by pressing on the control, only the hour, minute, and time of day can be changed.
+
+# [iOS\macOS](#tab/ios)
+
+<!-- David, maybe you can fill this out? -->
+
+# [Windows](#tab/windows)
+
+On Windows, the `Format` property doesn't affect what is displayed by the control. When the picker is shown by pressing on the control, only the hour, minute, and time of day can be changed. For more information about the Windows control, see [Time picker - Windows apps](/windows/apps/design/controls/time-picker).
```
Comment (davidbritch) [inline docs/user-interface/controls/timepicker.md]: Looking at the source code, `MapFormat` (https://github.com/dotnet/maui/blob/aba262370f299fe685250e3d1fa870f65f821a8a/src/Core/src/Handlers/TimePicker/TimePickerHandler.Windows.cs#L21) calls `UpdateTime` (https://github.com/dotnet/maui/blob/aba262370f299fe685250e3d1fa870f65f821a8a/src/Core/src/Platform/Windows/TimePickerExtensions.cs#L13) which says the format should switch between 24/12 hour clock.

Comment (adegeo) [reply docs/user-interface/controls/timepicker.md]: Good catch

**Comment by adegeo** [comment]
@davidbritch OK fixed. I looked at the code for mac/ios and it looks like ios is processing the string while macos isn't.

**Comment by davidbritch** [comment]
> @davidbritch OK fixed. I looked at the code for mac/ios and it looks like ios is processing the string while macos isn't.

I just tried it on iOS/Mac and you're correct. It's not really a picker control on Mac - just fields to enter the time.

**Review by davidbritch — APPROVED** [review]
LGTM

**Pull request merged**